src/index.js

```diff
@@ -186,7 +186,7 @@ class MarkdownShortcuts {
       },
       {
         name: 'asterisk-ul',
-        pattern: /^(\*|\+)\s$/g,
+        pattern: /^\+\s$/g,
```

Comment:
Can you add a note somewhere here that the asterisk was removed in quill version xx.xx.xx?

Thanks.

Reply:
I added a comment the line above.